x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/Literal.java

```diff
@@ -38,7 +38,11 @@ public Literal(Source source, Object value, DataType dataType) {
     public Literal(Source source, String name, Object value, DataType dataType) {
         super(source, name == null ? source.text() : name, emptyList(), null);
         this.dataType = dataType;
-        this.value = DataTypeConversion.convert(value, dataType);
+//        if (dataType == DATE) {
```

Comment:
?

Reply:
sorry, :-( this is again a left over from an attempt to use LocalDateTime for the `DATE` types.